src/main/java/de/tum/in/www1/artemis/service/notifications/GroupNotificationService.java

```diff
@@ -202,6 +202,7 @@ private void notifyGroupsWithNotificationType(GroupNotificationType[] groups, No
                 case DUPLICATE_TEST_CASE -> createNotification((Exercise) notificationSubject, author, group, NotificationType.DUPLICATE_TEST_CASE,
                         (String) typeSpecificInformation);
                 case ILLEGAL_SUBMISSION -> createNotification((Exercise) notificationSubject, author, group, NotificationType.ILLEGAL_SUBMISSION, (String) typeSpecificInformation);
+                case PLAGIARISM_CASE -> createNotification((Exercise) notificationSubject, author, group, NotificationType.PLAGIARISM_CASE, (String) typeSpecificInformation);
```

Comment:
Because there is no new method in this service and I assume from the logic the new notification only makes sense as a SingleUserNotification (1to1) notification. This added type to the switch case was added simply not to break the switch case/to please the compiler, but this does not really makes sense. I added a default case in this (soon to be merged) PR [#4335](https://github.com/ls1intum/Artemis/pull/4335)
![image](https://user-images.githubusercontent.com/65814168/143265245-5203c5c7-577b-428a-ba86-224f01aed7ae.png)

So please just remove this case altogether and add the default case from my PR.